### PR TITLE
Newbranch52

### PR DESCRIPTION
**Pull request by gudmdharalds**

### CODE REVIEW

File: bla-100.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
```
Comment (gudmdharalds) [inline bla-100.php]: VIP: Approved file.

File: bla-101.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
+
+echo get_post_meta( 100, 'data', true );
```
Comment (gudmdharalds-a8c) [inline bla-101.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_post_meta' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-101.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+echo get_post_meta( 100, 'data', true );
+echo get_post_meta( 100, 'data', true );
```
Comment (gudmdharalds-a8c) [inline bla-101.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_post_meta' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-101.php [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+echo get_post_meta( 100, 'data', true );
+
+echo get_post_meta( 100, 'data', true );
```
Comment (gudmdharalds-a8c) [inline bla-101.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_post_meta' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Pull request closed without merging**